src/main/scala/org/broadinstitute/dsde/agora/server/webservice/routes/MockAgoraDirectives.scala

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.dsde.agora.server.webservice.routes
+
+import org.broadinstitute.dsde.agora.server.AgoraConfig
+import org.broadinstitute.dsde.vault.common.directives.OpenAMDirectives
+import org.broadinstitute.dsde.vault.common.util.ImplicitMagnet
+import spray.routing.Directives._
+import spray.routing._
+
+import scala.concurrent.ExecutionContext
+
+trait MockAgoraDirectives extends AgoraDirectives {
+  def commonNameFromCookie(magnet: ImplicitMagnet[ExecutionContext]): Directive1[String] = {
+    OpenAMDirectives.commonNameFromCookie(magnet)
```

Comment:
I'm confused about why we are still getting the common name from OpenAM here - should we be grabbing a name from AgoraConfig instead?